var/spack/repos/builtin/packages/fox/package.py

```diff
@@ -37,6 +37,7 @@ class Fox(AutotoolsPackage):
     depends_on('libxi')
     depends_on('libxrandr')
     depends_on('gl', when='+opengl')
+    depends_on('mesa-glu', when='+opengl', type='link')
```

Comment:
I think the package should depend on `glu` (a virtual dependency like `gl`) rather than on a specific implementation.

Reply:
I renamed it to glu.